openapi/v2/device-sdk.yaml

```diff
@@ -1254,6 +1254,36 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /validate:
+    post:
+      summary: Run the protocol-specific validation for a Device's ProtocolProperties.
```

Comment:
```suggestion
      summary: Run the device-specific validation for a Device's Properties.
```

Reply:
Do we need to update route to `/validate/device` or not since we'll update Validator signature ?

Reply:
IMO, we still want `/validate/device` so it is explicit from a REST API perspective.

Reply:
also, we might want to validate provision watcher or other new objects one day.

Reply:
I've opened another PR in go-mod-core-contracts to update the route:
https://github.com/edgexfoundry/go-mod-core-contracts/pull/700